src/System.ComponentModel.TypeConverter/src/System/ComponentModel/LicenseManager.LicenseInteropHelper.cs

```diff
@@ -47,10 +46,19 @@ private class LicenseInteropHelper
             private static object AllocateAndValidateLicense(RuntimeTypeHandle rth, IntPtr bstrKey, int fDesignTime)
             {
                 Type type = Type.GetTypeFromHandle(rth);
-                CLRLicenseContext licensecontext = new CLRLicenseContext(fDesignTime != 0 ? LicenseUsageMode.Designtime : LicenseUsageMode.Runtime, type);
-                if (fDesignTime == 0 && bstrKey != (IntPtr)0)
+                string key = null;
+                if (bstrKey != IntPtr.Zero)
                 {
-                    licensecontext.SetSavedLicenseKey(type, Marshal.PtrToStringBSTR(bstrKey));
+                    key = Marshal.PtrToStringBSTR(bstrKey);
+                }
+                return AllocateAndValidateLicense2(type, key, (fDesignTime == 1 ? true : false));
+            }
```

Comment:
Nit: please either add a blank line between these functions, or make AllocateAndValidateLicense2 a local function.  I see below that you plan to delete all of this, but until then, it'd be nice to keep it tidy.